lib/agent.js

```diff
@@ -261,6 +263,21 @@ class Agent {
             agentVersion: this.config.version,
             licensed: this.config.licensed
         }
+        if (this.launcher?.readPackage) {
+            const { modules } = this.launcher.readPackage()
+            if (!semver.valid(modules['node-red'])) {
```

Comment:
should this be inverted Ben?

Reply:
```suggestion
            if (semver.valid(modules['node-red'])) {
```

I think the `!` is a left over

Reply:
No, it should have had 2 because the output is either null or a string, but the lint rules don't like that now.